doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -8,10 +8,19 @@ path: blob/master/doc/
 
     yum install epel-release
 
+Running with webmatic php
+
     rpm -Uvh https://mirror.webtatic.com/yum/el7/webtatic-release.rpm
 
     yum install composer cronie fping git httpd ImageMagick jwhois mariadb mariadb-server mtr MySQL-python net-snmp net-snmp-utils nmap php72w php72w-cli php72w-common php72w-curl php72w-gd php72w-mbstring php72w-mysqlnd php72w-process php72w-snmp php72w-xml php72w-zip python-memcached rrdtool
 
+Running with CentOS SCL php-fpm
+
+    yum install centos-release-scl
+    yum install rh-php72-php-fpm rh-php72-php-cli rh-php72-php-common rh-php72-php-curl rh-php72-php-gd rh-php72-php-mbstring rh-php72-php-process rh-php72-php-snmp rh-php72-php-xml rh-php72-php-zip rh-php72-php-memcached rh-php72-php-mysqlnd
```

Comment:
You are missing a lot of dependencies and have only included php.

Reply:
Ahh, didn't see that php and other dependencies where merged in the same yum statement.
Updated via https://github.com/deajan/librenms/commit/bfd33d9512e915cc7e48502fe3e2b503258c3771